libkineto/src/plugin/xpupti/XpuptiVersionLogger.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <level_zero/ze_api.h>
+#include <plugin/xpupti/XpuptiVersionLogger.h>
+#include <pti/pti_view.h>
+#include <sycl/version.hpp>
+namespace KINETO_NAMESPACE {
+namespace {
+
+std::string getLevelZeroVersion() {
+  uint32_t version = 0;
+  auto result = zeInit(0);
```

Comment:
Check if we need this `0` here.
```suggestion
  auto result = zeInit();
```